docs/reference/modules/node.asciidoc

```diff
@@ -89,93 +91,85 @@ memory and CPU in order to deal with the gather phase.
 ===============================================
 
 [[master-node]]
-==== Master-eligible node
+==== Manager node
 
-The master node is responsible for lightweight cluster-wide actions such as
+The manager node is responsible for lightweight cluster-wide actions such as
 creating or deleting an index, tracking which nodes are part of the cluster,
 and deciding which shards to allocate to which nodes. It is important for
-cluster health to have a stable master node.
+cluster health to have a stable manager node.
 
-Any master-eligible node that is not a <<voting-only-node,voting-only node>> may
-be elected to become the master node by the <<modules-discovery,master election
-process>>.
+Any control node that is not a <<voting-only-node,voting-only node>> can
+become the manager through the <<modules-discovery, election process>>.
 
-IMPORTANT: Master nodes must have access to the `data/` directory (just like
+IMPORTANT: Manager nodes must have access to the `data/` directory (just like
 `data` nodes) as this is where the cluster state is persisted between node
 restarts.
 
-[[dedicated-master-node]]
-===== Dedicated master-eligible node
+[[dedicated-control-node]]
+===== Dedicated control nodes
 
-It is important for the health of the cluster that the elected master node has
-the resources it needs to fulfill its responsibilities. If the elected master
-node is overloaded with other tasks then the cluster may not operate well. In
+It is important for the health of the cluster that the manager node has
+the resources it needs to fulfill its responsibilities. If the manager
+is overloaded with other tasks then the cluster may not operate well. In
 particular, indexing and searching your data can be very resource-intensive, so
 in large or high-throughput clusters it is a good idea to avoid using the
-master-eligible nodes for tasks such as indexing and searching. You can do this
-by configuring three of your nodes to be dedicated master-eligible nodes.
-Dedicated master-eligible nodes only have the `master` role, allowing them to
-focus on managing the cluster. While master nodes can also behave as
+control nodes for tasks such as indexing and searching. You can do this
+by configuring three of your nodes to be dedicated control nodes.
+Dedicated control nodes only have the `control` role, allowing them to
+focus on cluster management tasks. While control nodes can also behave as
```

Comment:
```suggestion
focus on controlling cluster management tasks. While control nodes can also behave as
```

I noticed this PR removed the verbiage around `manage` and `control. I wonder if this is because these verbs lack a clear delineation  

A `manager` node manages the cluster
A `control` node controls the cluster 

While the nouns `manager` and `control` have a much clearer delineation.

What does everyone think?

Reply:
I think we do need to provide a clearer description of what tasks the manager and control nodes perform. I was trying to avoid statements like "A manager node manages the cluster" that really don't provide any additional info.

This was just a quick pass to "try on" the terminology. There's definitely some fine-tuning to be done.